### PR TITLE
Created a full Wiki for Installing Unicorn

### PR DESCRIPTION
**Pull request by kennyparsons**
In this pull request, I created a wiki. Currently, the instruction set for the "UnicornSuite" is dispersed throughout the three repositories. This wiki brings them all under 1 cohesive guide for new/intermediate users. I also covered a few "gotcha" details that are discussed in a few of the closed issues, but not in the readme.

Changes:

Added wiki.md
Edited README.md to link to the wiki
Added an images folder with a screenshot

### CODE REVIEW

**Comment by Maxou44** [comment]
Can you review it @drouarb ?

File: Wiki.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+So, to install and configure UnicornTranscoder, complete the following steps:
+1. `cd /opt`
+2. `git clone https://github.com/UnicornTranscoder/UnicornTranscoder.git`
+3. `cd UnicornTranscoder`
+4. `npm install`
+5. `npm run install`
+
+The transcoder is now ready to run. Just like the loadbalancer, we're going to run this as a service. Be sure to substitute the envrionment variables for your own setup/FQDN:
+1. `nano /etc/systemd/system/unicorntranscoder.service`
```
Comment (drouarb) [inline Wiki.md:201]: Maybe names of services should be consistent earlier UnicornLoadBalancer was camelcase

File: Wiki.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+After=network.target
+
+[Service]
+Restart=on-abnormal
+Type=simple
+
+User=root
+Group=root
+
+Environment=TRANSCODER_DEBUG=true
```
Comment (drouarb) [inline Wiki.md:215]: This shouldn't be enabled by default, it produces a lot of log

File: Wiki.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+3. `systemctl start unicorntranscoder.service`
+4. `systemctl status unicorntranscoder.service`
+
+If the output of of the status command shows active, the next step is to set up nginx for the transcoder. You can find examples of an Nginx configuration here: https://github.com/UnicornTranscoder/Nginx. 
+
+\* Note:
+If you want to adjust the envrionment variables in `config.js`, make sure you re-run the npm build with `npm install` & `npm run install`. If you opt to override these variables in the service (like we did above), edit the service file. If you edit a service after it has already been loaded, you need to reload the daemon with `systemctl daemon-reload` & `systemctl restart unicorntranscoder.service`. As a rule of thumb, if you use service environment variables, don't edit the config.js variables (or vice versa). Keep all your variable definitions in the same place. 
+
+Once the FQDN is up and active with working SSL, point your browser to it. It will look something like this:
+![](/images/UnicornTranscoder.png?raw=true)
```
Comment (drouarb) [inline Wiki.md:247]: Maybe we could store the image somewhere else than the code repository ?

**Review by drouarb — COMMENTED** [review]
@Maxou44 double check ?

File: Wiki.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+The next step is to add a media library (or edit if you've already configured libraries using the old `/media` path). Once the library is refreshed and the media is available in Plex, we can move on to the next step. 
+
+-----
+
+#### Configure UnicornLoadBalancer
+In this guide, we'll be installing all unicorn products into `/opt`. We'll also be running commands as root, but as a best practice, don't run as root in production. Using root allows us to learn the process. Once a full knowledge of the suite is achieved, go back and run unicorn with the least amount of privileges. 
+Required Software:
+* NPM/Nodejs
+* git
+You can run Unicorn with Redis, but for a small plex setup with a handful of user/shared accounts, using the built-in sqlite.db will suffice.
```
[the remark this answers is not in the document]
Comment (kennyparsons) [reply Wiki.md:55]: Thanks for the clarification.

File: Wiki.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+6. UnicornFFMPEG sends the arguments to the UnicornLoadBalancer
+7. UnicornTranscoder pulls the FFMPEG arguments from UnicornLoadBalancer
+8. UnicornTranscoder launches FFMPEG and serves the request for the stream directly to the client.
+
+As you can see, there is no limit to the amount of "nodes" you can add to the transcoding cluster. The loadbalancer distributes the transcoding jobs across all of the nodes seamlessly. Another benefit is the stream itself is served from the node itself (UnicornTranscoder). So you can even use nodes to distribute your plex traffic and take the load off your home network bandwidth. 
+
+#### Assumptions
+1. You have working linux Plex Server or can set up one quickly with no further instruction. In this guide, we'll be covering PMS deployed in a container. 
+2. You have a working reverse proxy with SSL. In this guide, we'll be using nginx.
+3. All nodes (UnicornTranscoders) have access to the source media files. This can be accomplish several different ways. Most people will use NFS, SMB, iSCSI, or some other network filesystem protocol tool (such as rclone).
```
[the remark this answers is not in the document]
Comment (kennyparsons) [reply Wiki.md:29]: @drouarb Does that option mean the loadbalencer will serve the raw media files to the node, the node transcodes, and then the node serves the stream?

File: Wiki.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+    * This will override the default localhost:port option with the FQDN of the loadbalancer. This environment variable is required for a working setup. 
+    * After this npm command finishes, a new Plex Transcoder binary will be saved to the ./bin folder of the repository. Copy the linux binary to your plex docker folder (or anywhere you want). In this example we're copying it to `/root/docker/plexv2/binaries/Plex Transcoder-linux`.
+
+In order to replace the built-in transcoder with this new one, we need to mount it into our plex container. Simply add the following mount to your plex container:
+```
+- "/root/docker/plexv2/binaries/Plex Transcoder-linux:/usr/lib/plexmediaserver/Plex Transcoder"
+```
+where `/usr/lib/plexmediaserver` is the binary location. Most mainstream plex containers use this path, such as the container released by LinuxServier.io. 
+
+With that new volume/file mount, restart plex to replace the built-in transcoder with the "fake" transcoder that interprets the transcode arguments.
```
[the remark this answers is not in the document]
Comment (kennyparsons) [reply Wiki.md:184]: @drouarb We need to restart the container to accept the new volume mount. Otherwise, when plex does call the transcoder, it's still going to have the built in one. Up to this point, the container still has the original. We have not replaced it. So perhaps we can reword this to "restart/run the container with the new volume mount definition"

File: Wiki.md (hunk 1 above)
Comment (kennyparsons) [reply Wiki.md:201]: This is the result of a "Find and Replace" to fix a spelling issue. Easy enough fix

File: Wiki.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kennyparsons) [reply Wiki.md:29]: According to the docs:
```
CUSTOM_DOWNLOAD_FORWARD: Enable or disable 302 for download links and direct play, if enabled, transcoders need to have access to media files. Consider changing to true for your needs.
```
This implies that this option only deals with downloads and direct play (i.e. a direct play where the unicorn software is not normally used will now use the transcoder node to serve the stream). If that's not correct, the docs could use a refresher.

File: Wiki.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (drouarb) [reply Wiki.md:184]: Oh yes, didn't understood that you were talking about a docker setup

File: Wiki.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Maxou44) [reply Wiki.md:29]: This parameter changes how transcoders access to the media file (for download, direct play, media optimizer and transcoded streams).

If enabled, transcoders access to files on the same mount point, they will read files from theirs own filesystem. FFmpeg will get a local file path.

If disabled (by default) each media access (transcode, optimization...) will call the load balancer to get the file. FFmpeg will get a HTTP link to stream the file using your load balancer.

We recommend to set it to false if your media files are stored on your load balancer server, if you mounted your media files from another service(using FUSE or a dedicated storage server, we recommend to set it to true and mount it on each trasncoders (you we save your load balancer bandwidth)
